packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md

```diff
@@ -21,17 +21,17 @@ This page provides the API rate limits for management activities, which is part
-| **Get System Log data:**<br>`/api/v1/logs`                                                                                         | 20               | 25               | 25      | 50         | 120                   |
-| **Get System Log data:**<br>`/api/v1/events`                                                                                       | 20               | 25               | 25      | 50         | 100                   |
-| **OAuth2 client configuration requests:**<br>`/oauth2/v1/clients`                                                                  | 25               | 50               | 50      | 100        | 100                   |
-| **Most other API actions:**<br>`/api/v1`                                                                                           | 100              | 300              | 300     | 600        | 1,200                 |
-| **Get a user by ID or sign in**<br>Only `GET` to `/api/v1/users/${idOrLogin}`                                                       | 100              | 300              | 300     | 1,000      | 2,000                 |
-| `/api/v1/certificateAuthorities`                                                                                                   | 100              | 100              | 100     | 150        | 100                   |
-| `/api/v1/devices`                                                                                                                  | 100              | 100              | 100     | 150        | 100                   |
+| **Get, update, or delete a group by ID:**<br>`/api/v1/groups/${id}` only                                                            | 100              | 600              | 600     | 600        | 1,000                 |
+| **Create or list users:**<br>Only `GET` or `POST` to `/api/v1/users`                                                               | 100              | 600              | 600     | 600        | 600                   |
+| **Update or delete a user by ID:**<br>Only `POST`, `PUT` or `DELETE` to `/api/v1/users/${id}`                                       | 100              | 600              | 600     | 600        | 600                   |
+| **Get System Log data:**<br>`/api/v1/logs`                                                                                         | 20               | 50               | 50      | 50         | 120                   |
+| **Get System Log data:**<br>`/api/v1/events`                                                                                       | 20               | 50               | 50      | 50         | 100                   |
+| **OAuth2 client configuration requests:**<br>`/oauth2/v1/clients`                                                                  | 25               | 100               | 100      | 100        | 100                   |
```

Comment:
Do we need to change "OAuth2" to "OAuth 2.0"?

Reply:
I did a search in dev docs and noticed that when they're in an endpoint, they're `/oauth2`.